pyface/api.py

```diff
@@ -112,7 +112,7 @@
 from .layered_panel import LayeredPanel
 from .message_dialog import error, information, warning, MessageDialog
 from .progress_dialog import ProgressDialog
-
+from .array_image import ArrayImage
```

Comment:
The imports are sorted alphabetically in this file so can you please update the location of this import statement?